internal/build/custom_builder_test.go

```diff
@@ -270,11 +271,12 @@ func TestEnvVars(t *testing.T) {
 	sha := digest.Digest("sha256:11cd0eb38bc3ceb958ffb2f9bd70be3fb317ce7d255c8a4c3f4af30e298aa1aab")
 	f.dCli.Images["localhost:1234/foo_bar:tilt-build-1551202573"] = types.ImageInspect{ID: string(sha)}
 	cb := f.customBuild(strings.Join(script, "\n"))
+	cb.Env = []string{"EXTRA=value"}
 	_, err := f.Build(refSetWithRegistryFromString("foo/bar", TwoURLRegistry), cb, nil)
 	require.NoError(t, err)
 }
 
-func TestEnvVars_ConfigRefWithLocalRegistry(t *testing.T) {
+func TestCustomBuildEnvVars_ConfigRefWithLocalRegistry(t *testing.T) {
```

Comment:
I renamed these two test cases to match the naming convention in the rest of the file.